translations/japanese/core-commands.ts

```diff
@@ -0,0 +1,132 @@
+		"User ${targetUsername} is offline.": "ユーザー名:${targetUsername}は見つかりませんでした。",
+		"The user \"${targetUsername}\" was not found.": "ユーザー名:\"${targetUsername}\"は見つかりませんでした。",
+		"The room \"${target}\" was not found.": "チャットルーム:\"${target}\"は見つかりませんでした。",
+		"You do not have permission to invite people into this room.": "あなたはこの部屋でユーザーを招待する権限を持っていません。",
+		"This user is already in \"${targetRoom.title}\".": "このユーザーはすでに\"${targetRoom.title}\"にいます。",
+		"Setting status messages in /busy is no longer supported. Set a status using /status.": "/busyでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"Setting status messages in /away is no longer supported. Set a status using /status.": "/awayでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"User '${target}' not found.": "ユーザー名'${target}'は見つかりませんでした。",
+		"${targetUser.name} does not have a status set.": "${targetUser.name}はステータスメッセージを設定していません。",
+		"${targetUser.name}'s status \"${targetUser.userMessage}\" was cleared by ${user.name}${displayReason}": "${targetUser.name}のステータスメッセージは${user.name}によって消去されました。${displayReason}",
+		"You don't have a status message set.": "ステータスメッセージを設定していません。",
+		"You have cleared your status message.": "ステータスメッセージを消去しました。",
+		"This user has not played any ladder games yet.": "このユーザーはladderが有効なバトルをしていません。",
+		"W[TN: initial for Wins]": "'Wins'の頭文字",
+		"L[TN: initial for Losses]": "'Losses'の頭文字",
+		"You already have the temporary symbol '${group}'.": "あなたはすでに一時的なランクである'${group}'になっています。",
+		"You must specify a valid group symbol.": "有効なランクの記号を特定する必要があります。",
+		"You may only set a temporary symbol below your current rank.": "現在のランクより下の一時的なシンボルしか設定できません。",
+		"Your temporary group symbol is now": "あなたの一時的なランクの記号",
+		"Currently, you're viewing Pokémon Showdown in ${language}.": "現在、Pokémon Showdownを${languege}で表示しています。",
+		"Valid languages are: ${languages}": "有効な言語一覧: ${langueges}",
+		"Pokémon Showdown will now be displayed in ${language} (except in language rooms).": "Pokémon Showdownは言語部屋以外の場所は${languege}で表示されます。",
+		"Note that rooms can set their own language, which will override this setting.": "言語部屋ではそれぞれが独自の言語を設定でき、その場合はユーザー設定より部屋の設定が優先されます。",
+		"/updatesettings expects JSON encoded object.": "/updatesettings を使用するには、JSONに変換されたファイルが必要です。",
+		"Unable to parse settings in /updatesettings!": "/updatesettings において、設定に使用するJSONを解析することができません。",
+		"Must be in a battle.": "バトルの中でなければいけません。",
+		"User ${target} not found.": "ユーザー ${target} は見つかりませんでした。",
+		"Must be a player in this battle.": "プレイヤーである必要があります。",
+		"${targetUser.name} has not requested extraction.": "${targetUser.name} は解析を要求していません。",
+		"You have already consented to extraction with ${targetUser.name}.": "あなたはすでに${targetUser.name}と解析に同意しています。",
+		"${user.name} consents to sharing battle team and choices with ${targetUser.name}.": "${user.name}と${targetUser.name}はバトルチームや技の選択を共有することに同意します。",
```

Comment:
I didn't using "\\" and "/". what's wrong?

Reply:
Wrong file.

If you look at the "Files" tab of the pull request, it will tag the line with the problem:

![image](https://user-images.githubusercontent.com/551184/147973011-c99d9a4c-d131-4214-a542-bf1e74645408.png)

Reply:
https://github.com/smogon/pokemon-showdown/pull/8608/files#diff-55d32fb17e511fd7e67bbf30149433ac0dedf961f847c65205ff39e82505137eL164

Reply:
oh, main.ts!
thank you.